integration/ui/satellite/signup_personal_user_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/input"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/integration/ui/uitest"
+	"storj.io/storj/private/testplanet"
+)
+
+func TestPersonalUserCanSignUp(t *testing.T) {
+	uitest.Run(t, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		signupPageURL := planet.Satellites[0].ConsoleURL() + "/signup"
+		fullName := "John Doe"
+		emailAddress := "test@email.com"
+		password := "qazwsx123"
+		page := browser.Timeout(10 * time.Second).MustPage(signupPageURL)
```

Comment:
I think that we don't need `Timeout(10 * time.Second)`, because in [run.go](https://github.com/storj/storj/blob/5f901c806658ed5ecc28ddfa4082444d0aadf226/integration/ui/uitest/run.go#L78) we already set `Timeout(time.Minute)`